.github/workflows/trigger-external-bindings.yml

```diff
@@ -0,0 +1,27 @@
+name: Trigger External Bindings Workflow
+on:
+  workflow_dispatch:
+  push:
+    paths:
+      - 'bdk-ffi/**'
+
+permissions: {}
+
+jobs:
+  build:
+    runs-on: ubuntu-24.04
+    steps:    
+    #Trigger bdk-jvm-test in bdk-jvm repository
+      - name: Trigger bdk-jvm-test in bdk-jvm repository
+        if: ${{ (github.ref_name == 'main' || github.ref_type == 'tag') }}
+        run: |
+          response=$(curl -X POST \
+            -H "Accept: application/vnd.github+json" \
+            -H "Authorization: token ${{ secrets.REPO_ACCESS_TOKEN }}" \
+            https://api.github.com/repos/bdk/bdk-jvm/dispatches \
```

Comment:
I'm playing around with this and one issue I found was that this line is incorrect. The url would be

```sh
https://api.github.com/repos/bitcoindevkit/bdk-jvm/dispatches
```